src/Jackett.Common/Definitions/exitorrent-org.yml

```diff
@@ -94,8 +94,6 @@ search:
     hashTag: 0
     # 0 all, 1 active, 2 dead
     incldead: 0
-    # 0 all, 1 polish, 5 dubbed, 6 subs, 2 non-polish, 3 freeleech, 4 highlighted
-    polish: "{{ if .Config.freeleech }}3{{ else }}1{{ end }}"
```

Comment:
Is your change intended to return ALL results which is to say, not just ones that have the Polish language? In which case setting the 1 to 0 on that line would achieve that?

Also, what about searching for freeleech now. Is that no longer option 3 on the polish parameter?